ddl/ddl_worker.go

```diff
@@ -111,19 +111,7 @@ func (d *ddl) finishDDLJob(t *meta.Meta, job *model.Job) (err error) {
 	switch job.Type {
 	case model.ActionDropSchema, model.ActionDropTable, model.ActionTruncateTable, model.ActionDropIndex:
 		if job.Version <= currentVersion {
```

Comment:
Both are OK. I prefer `<=` because I think it's more clear:
```go
if version <= currentversion {
    if version == someSpecifiedVersion {
        // do special logic.
    } else if version == specialVersion2 {
        // do special logic.
    }
    // do some downstream logic.
} else {
    // Error here
}
```